src/main/java/com/twilio/oai/Inflector.java

```diff
@@ -84,6 +84,7 @@ private void initIrregulars() {
              irregulars.add("BuildStatus");
              irregulars.add("DialingPermissions");
              irregulars.add("Media");
+             irregulars.add("GoodData");
```

Comment:
We should be careful about adding exceptional cases. Looking into alternatives ...

Reply:
https://github.com/twilio/twilio-oai-generator/pull/332